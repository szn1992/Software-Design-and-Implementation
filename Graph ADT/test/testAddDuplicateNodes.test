# Test adding duplicated nodes, there should not be
# any duplicate nodes in graph after adding

# Create a graph
CreateGraph graph1

# Add duplicate nodes
AddNode graph1 n1
AddNode graph1 n1
AddNode graph1 n1

# Print the nodes in the graph
ListNodes graph1