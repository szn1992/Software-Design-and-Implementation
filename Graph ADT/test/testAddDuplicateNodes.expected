# Test adding duplicated nodes, there should not be
# any duplicate nodes in graph after adding

# Create a graph
created graph graph1

# Add duplicate nodes
added node n1 to graph1
added node n1 to graph1
added node n1 to graph1

# Print the nodes in the graph
graph1 contains: n1